benchmark/Snakefile

```diff
@@ -0,0 +1,537 @@
+              <(grep eps_an results/polyDFE/{wildcards.seeds}/{wildcards.chrms}/pop{wildcards.ids}/pop{wildcards.ids}.polyDFE.results -A 3 | tail -1 | awk 'BEGIN{{OFS="\t"}}{{print $2,$3}}') \
+              <(grep eps_an results/polyDFE/{wildcards.seeds}/{wildcards.chrms}/pop{wildcards.ids}/pop{wildcards.ids}.polyDFE.results -A 1 | tail -1 | awk '{{print $3}}') > {output}
+        """
+
+rule combine_polydfe_bestfits:
+    input:
+        pop0_res = expand(output_dir + "/polyDFE/{seeds}/{chrms}/pop0/pop0.polyDFE.bestfit", seeds=seed_array, chrms=chrm_list),
+        pop1_res = expand(output_dir + "/polyDFE/{seeds}/{chrms}/pop1/pop1.polyDFE.bestfit", seeds=seed_array, chrms=chrm_list),
+        pop2_res = expand(output_dir + "/polyDFE/{seeds}/{chrms}/pop2/pop2.polyDFE.bestfit", seeds=seed_array, chrms=chrm_list)
+    output:
+        expand(output_dir + "/polyDFE/{chrms}/pop{ids}.polyDFE.bestfits", chrms=chrm_list, ids=[0, 1, 2])
+    threads: 1
+    shell:
+        """
+        cat {input.pop0_res} | sed '1i#likelihood\tS_d\tb\ttheta_bar' > {output[0]}
+        cat {input.pop1_res} | sed '1i#likelihood\tS_d\tb\ttheta_bar' > {output[1]}
+        cat {input.pop2_res} | sed '1i#likelihood\tS_d\tb\ttheta_bar' > {output[2]}
+        """
+
+#rule plot_polydfe_results
+
+# ###############################################################################
+# DFE-alpha
+# ###############################################################################
+
+rule run_dfe_alpha:
+    input:
+        output_dir + "/dfe-alpha/{seeds}/{chrms}/pop{ids}/pop{ids}.dfe-alpha.neu.config",
+        output_dir + "/dfe-alpha/{seeds}/{chrms}/pop{ids}/pop{ids}.dfe-alpha.nonneu.config",
+        output_dir + "/dfe-alpha/{seeds}/{chrms}/pop{ids}/pop{ids}.dfe-alpha.fs"
+    output:
+        output_dir + "/dfe-alpha/{seeds}/{chrms}/pop{ids}/nonneu/est_dfe.out"
+    threads: 1
+    resources: time_min=60, mem_mb=5000, cpus=1
+    shell:
+        "est_dfe -c {input[0]}; est_dfe -c {input[1]}"
+
+#rule plot_dfe_alpha_results
+
+# ###############################################################################
+# anavar
+# ###############################################################################
+
+#rule run_anavar:
+#    input:
+#        "results/anavar/pop0.anavar.ctl.txt"
+#    output:
+#        "results/anavar/pop0.anavar.results",
+#        "results/anavar/pop0.anavar.log"
+#    threads: 1
+#    shell:
+#        "anavar {input[0]} {output[0]} {output[1]}"
+
+
+# ###############################################################################
+# prfreq
+# ###############################################################################
+
+#rule run_prfreq:
+#    input:
+#    output:
+#    threads: 1
+#    shell:
+
+
+# ###############################################################################
+# plots
```

Comment:
it would be great @xin-huang if all of these plots could go into their own module. see the `plots.py` file in the original analysis repo for an example